force-app/main/default/classes/MA_ReportSourceBatchableTest.cls

```diff
@@ -246,10 +248,11 @@ private class MA_ReportSourceBatchableTest {
             // because the batchable throws the error in the start method
             // then we're getting a synchronous exception caught rather than
             // the job's finish method logging the error asynchronously
-            System.assert( ex.getMessage().containsIgnoreCase( 'The data you’re trying to access is unavailable' ), 'failed for wrong reason' );
-
+            noDataFoundExceptionThrown = true;
         }
 
+        System.assert(noDataFoundExceptionThrown);
```

Comment:
Thanks for the language fixes in this test class, and for [contributing](https://github.com/douglascayers-org/sfdx-mass-action-scheduler/blob/master/CONTRIBUTING.md) to Mass Action Scheduler. 

For the new assert at line 254, for consistency with existing code style, please add a single space after ( and before ) in the method call. 

Additionally, I think there’s still value in the assert having the second argument, 'failed for wrong reason', to provide context around the failure.

Reply:
I forgot the code style, sorry. I added the message and also another catch for generic exceptions (they shouldn't be catched, so I added an assertion that will fail inside of it). This way we can catch both scenarios where no exception is thrown and the wrong exception is thrown.

Reply:
Thanks, I'm testing the PR and if all is well will merge it in :)